client/lib/post-normalizer/utils.js

```diff
@@ -30,6 +30,16 @@ export function imageSizeFromAttachments( post, imageUrl ) {
 	}
 }
 
+export function stripPhotonParams( imageUrl ) {
+	if ( ! imageUrl ) {
+		return imageUrl;
+	}
+	const parsedUrl = url.parse( imageUrl );
+	parsedUrl.query = parsedUrl.search = ''; // remove query params
```

Comment:
Do you think there's a case for us preserving certain Photon params, e.g. for a percentage-based crop or filters? 

https://developer.wordpress.com/docs/photon/